src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -513,7 +525,9 @@ void buildOverrideHearingLocations() {
     }
 
     void enableAdjournmentFlag() {
```

Comment:
This method does more than enable the feature flag. It would be better to keep them separate. Also better to just remove the helper method entirely and have the given in each test.

Reply:
basically every time we need to enable the flag we need to be in progress so its better together. Having this as a method is better than repeating itself in each test where its needed

Reply:
see my comment